pages/docs/doc/ibge.json

```
@@ -217,6 +217,123 @@
                     }
                 }
             }
+        },
+        "/ibge/nomes/v1/{nome}": {
+            "get": {
+                "tags": ["IBGE"],
+                "summary": "Busca a frequencia de um determinado nome",
+                "description": "Um ou mais nomes delimitados pelo caracter | (pipe)",
+                "parameters": [
+                    {
+                        "name": "sexo",
+                        "description": "Filtrar pelo sexo, cujos valores podem ser M (Masculino) ou F (Feminino)\n",
+                        "in": "query"
+                    },
+                    {
+                        "name": "groupBy",
+                        "description": "Caso deseje obter a frequência por algum nível geográfico, informe o parâmetro groupBy\n",
+                        "in": "query"
+                    },
+                    {
+                        "name": "localidade",
+                        "description": "Caso deseje obter a frequência referente a uma dada localidade\n",
+                        "in": "query"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/NameFrequency"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "O nome não foi encontrado",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ErrorMessage"
+                                },
+                                "example": {
+                                    "name": "NotFoundError",
+                                    "message": "UF não encontrado",
```

A mensagem de erro está correta? Ou seria
```suggestion
                                    "message": "Nome não encontrado.",
```